src/code.c

```diff
@@ -212,7 +212,7 @@ void SET_ENDLINE_BODY(Obj body, Obj val)
 ** Fill in filename and line of a statement, checking we do not overflow
 ** the space we have for storing information
 */
-Stat fillFilenameLine(Int fileid, Int line, Int size, Int type)
+static Stat fillFilenameLine(Int fileid, Int line, Int size, Int type)
 {
```

Comment:
Nitpick, and not really anything to do with this PR, but why is the function name not capitalised?